pkg/extensions/search/search_test.go

```diff
@@ -1798,110 +1800,87 @@ func TestExpandedRepoInfo(t *testing.T) {
 
 func TestUtilsMethod(t *testing.T) {
```

Comment:
This should be split into multiple tests, and those be moved to the appropriate packages. For example the annotation tests should be in the `convert` package.